src/aiida/cmdline/params/options/__init__.py

```diff
@@ -63,17 +69,27 @@
     'GROUP_CLEAR',
     'HOSTNAME',
     'IDENTIFIER',
+    'INCLUDE_ATTRIBUTES',
```

Comment:
I am not stoked that we're exporting all of these, but this is already a mess so I guess it doesn't matter.